### ISSUE TITLE
After running the batch file and trying to open localhost:8000, it just brings me to a directory page

### ISSUE THREAD

**Issue by Manchovies** [space-nuko/ComfyBox#136]
Expected behavior is that it'll load ComfyBox like it used to. Not sure what I changed, if anything, to make this happen instead of opening ComfyBox, but I start ComfyUI and when it's loaded, I run "run.bat" and in the command prompt window, it tells me the server is running on localhost:8000, but when I open it, this is what I get: 
![image](https://github.com/space-nuko/ComfyBox/assets/113330032/279e1cdb-64b2-4d3f-bd7c-6fb00eed9a86)

**Comment by hobojoker** [comment]
Try manually navigating to your local ip address port 8000

**Comment by Manchovies** [comment]
I get the same results if I do that. Thanks for the response! If there’s anything else I can try or info I can provide please let me know

**Comment by hobojoker** [comment]
Hmm, is there a reason you're going to localhost:8001 instead of localhost:8000?

**Comment by Manchovies** [comment]
Good catch! I edited serve.py to use port 8001 just in case 8000 was taken and I didn’t know it. I get the same issue both ways, with 8000 and 8001.

**Comment by hobojoker** [comment]
Did you by chance change the IP also for your comfyui?

**Comment by Manchovies** [comment]
Nope, just changed the port. And just for curiosity’s sake, if I changed the IP, I’m guessing it wouldn’t let me get through to this directory page. Is that right?

**Comment by lilshippo** [comment]
Getting the exact same issue, just freshly installed it. although mine shows 8000

**Comment by Katmoget** [comment]
I have the same issue, managed to get around by doing this every time I launch ComfyBox : https://github.com/space-nuko/ComfyBox/issues/132#issuecomment-1663472749
